scripts/commands/update-translations.js

```diff
@@ -0,0 +1,170 @@
+import { exec } from 'child_process';
+import { promisify } from 'util';
+import https from 'https';
+import fs from 'fs';
+import { error, success, info, warning, gitFactory } from '../utils.js';
+
+const execPromise = promisify( exec );
+const REQUIRED_TRANSLATION_PERCENTAGE = 85;
+const POT_FILE_PATH = 'languages/wc-calypso-bridge.pot';
+
+async function downloadTranslations( url, outputPath ) {
+	return new Promise( ( resolve, reject ) => {
+		const file = fs.createWriteStream( outputPath );
+
+		https
+			.get( url, ( response ) => {
+				response.pipe( file );
+				file.on( 'finish', () => {
+					file.close();
+					resolve();
+				} );
+			} )
+			.on( 'error', ( err ) => {
+				reject( err );
+			} );
+	} );
+}
+
+async function verifyLangData( langData ) {
+	try {
+		const parsedData = JSON.parse( langData );
+		if (
+			! parsedData.translation_sets ||
+			! Array.isArray( parsedData.translation_sets )
+		) {
+			error( 'Invalid language data format.' );
+			process.exit( 1 );
+		}
+	} catch ( err ) {
+		error( 'Invalid language data format.' );
+		process.exit( 1 );
+	}
+}
+
+async function updateTranslations() {
+	const git = gitFactory();
+
+	try {
+		info( 'Executing wp i18n make-pot command...' );
+
+		const makePotResult = await execPromise(
+			`wp i18n make-pot . ${ POT_FILE_PATH } --ignore-domain`
```

Comment:
Currently `.gitignore` has `languages/*` in it, which would be removed.

Reply:
Done